get_s3_test.go

```diff
@@ -31,7 +31,7 @@ func TestS3Getter(t *testing.T) {
 
```

Comment:
We should probably remove the [`init`](https://github.com/hashicorp/go-getter/blob/d305097eb8901ba73041f9b4efcc8eae90f15dcd/get_s3_test.go#L12-L22) from this file, and document the expectation for how these tests should run (only in CI from PRs). This will be useful in the future, where external contributors will not be able to run these tests locally anymore, for example.
https://github.com/hashicorp/go-getter/blob/d305097eb8901ba73041f9b4efcc8eae90f15dcd/get_s3_test.go#L12-L22